src/pages/docs/administration/reporting/report-on-deployments-using-excel.md

```diff
@@ -7,11 +7,7 @@ description: How to report on deployments using Excel & XML
 navOrder: 10
 ---
 
-:::div{.hint}
```

Comment:
Scope creep: Do we need to adjust the title? It looks like it's being escaped or something
<img width="1238" alt="image" src="https://github.com/OctopusDeploy/docs/assets/108677358/e0e42082-37a4-4edc-b768-d3c70e2f27a1">

Reply:
Changed to "and"
![image](https://github.com/OctopusDeploy/docs/assets/11970672/a663759a-3aea-4167-bec2-04deab6ac75a)